metricbeat/module/elasticsearch/ml_job/_meta/data.json

```diff
@@ -1,33 +1,101 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "agent": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
     "elasticsearch": {
         "cluster": {
-            "id": "3LbUkLkURz--FR-YO0wLNA",
-            "name": "es1"
+            "id": "8l_zoGznQRmtoX9iSC-goA",
+            "name": "docker-cluster"
         },
         "ml": {
             "job": {
                 "data_counts": {
```

Comment:
One thing that is confusing me is that in this `data.json` here, the field path is `elasticsearch.ml.job.data_counts.*`. But if I look at `fields.yml`, I'm seeing `elasticsearch.ml.job.data.counts.*`. Shouldn't the two match up or, failing that, shouldn't there be aliases from all the `elasticsearch.ml.job.data_counts.*` fields to the corresponding `elasticsearch.ml.job.data.counts.*` fields so there are breaking changes?

Reply:
Yes, you're right. I want a pair of eyes like yours to spot such things :smile: